docs/user/guides/technical-docs-seo-guide.rst

```diff
@@ -92,30 +81,47 @@ which we'll discuss shortly,
 the most important thing is making sure the spider can follow links on your site
 and get to all your pages.
 
-Avoid orphan pages
-~~~~~~~~~~~~~~~~~~
+Avoid orphan pages ✅️
+~~~~~~~~~~~~~~~~~~~~~~
 
 Sphinx calls pages that don't have links to them "orphans"
 and will throw a warning while building documentation that contains an orphan
 unless the warning is silenced with the :ref:`orphan directive <sphinx:metadata>`:
 
-::
+.. tabs::
 
-    $ make html
-    sphinx-build -b html -d _build/doctrees . _build/html
-    Running Sphinx v1.8.5
-    ...
-    checking consistency... /path/to/file.rst: WARNING: document isn't included in any toctree
-    done
-    ...
-    build finished with problems, 1 warning.
+   .. tab:: Sphinx
 
-You can make all Sphinx warnings into errors during your build process
-by setting ``SPHINXOPTS = -W --keep-going`` in your Sphinx Makefile.
+        Sphinx will warn you when a page isn't referenced by the remaining documentation.
+        We recommend failing your builds whenever Sphinx warns you,
+        using the ``fail_on_warnings`` option in :ref:`.readthedocs.yaml <config-file/v2:sphinx>`.
 
+        Here is an example of a warning of an unreferenced page::
 
-Avoid uncrawlable content
-~~~~~~~~~~~~~~~~~~~~~~~~~
+            $ make html
+            sphinx-build -b html -d _build/doctrees . _build/html
+            Running Sphinx v1.8.5
+            ...
+            checking consistency... /path/to/file.rst: WARNING: document isn't included in any toctree
+            done
+            ...
+            build finished with problems, 1 warning.
+
+   .. tab:: MkDocs
+
+      MkDocs automatically includes all ``.md`` files in the main navigation 💯️.
+      So it does not need to report back about unreferenced files.
+      You have to go out of your way and install a plugin,
+      if you want to hide files from the navigation.
+      If you do want to scan your documentation for unreferenced files and images,
+      a plugin like `mkdocs-unused-files`_ does the job.
```

Comment:
```suggestion
      MkDocs automatically includes all ``.md`` files in the main navigation 💯️.
      This makes sure that all files are discoverable by default,
      however there are configurations that allows for issues to arise.
      If you want to scan your documentation for unreferenced files and images,
      a plugin like `mkdocs-unused-files`_ does the job.
```

This was needlessly snarky. `go out of your way` is not language we generally want to use. If I'm a user landing on this page, and doing something like this, I'm going to feel judged.

Reply:
Thanks! The language should always be inclusive, thanks for fixing it directly :+1: